### DISCUSSION TITLE
Easy to use

### DISCUSSION THREAD

**Discussion by markusthiel** [Cockpit-HQ/Cockpit#128] [General]
Hello everyone!

Previously, Cockpit CMS was an ideal method for me to quickly create a CMS from an existing PHP website. 
However, it seems to have become complicated now, or am I mistaken? Back then, I would install the system and replace the content part in the PHP website with a snippet that would fetch the content from the CMS. Is this possibility still available easily, or is it currently running differently? I couldn't find an answer to this in the documentation either. Actually, all I would like are some examples of how content is delivered without having to learn a new language.

**Comment by aheinze** [comment]
Hey 👋

Can you provide some code examples for your snippets, which gives some context?
